tools/pipelines/templates/upload-dev-manifest.yml

```
@@ -28,7 +31,7 @@ steps:
     workingDirectory: ${{ parameters.buildDirectory }}
     script: |
       mkdir upload_release_reports
-      flub release report-unreleased --version $(SetVersion.version) --fullReportFilePath generate_release_reports/manifest.full.json --outDir upload_release_reports
+      flub release report-unreleased --version $(SetVersion.version) --fullReportFilePath generate_release_reports/manifest.full.json --outDir upload_release_reports --branchName ${{ parameters.branchName }}
```

It's somewhat subjective but since it's a system variable in ADO, I'd recommend using it directly here and removing the new parameter to the template (and the line where we pass it). Mostly my motivation is to avoid the use of `variables.<something>` with expression-template syntax `${{}}`, aka "at compile time" (doesn't happen in this file, but in `include-set-package-version.yml` in order to pass it to this file). In this particular scenario it would work because Build.SourceBranch is a system variable that ADO makes available at compile time, but we've had bugs in the past trying to use our own variables with that syntax when they might not exist yet. So I lean towards avoiding that combination whenever possible.
```suggestion
      flub release report-unreleased --version $(SetVersion.version) --fullReportFilePath generate_release_reports/manifest.full.json --outDir upload_release_reports --branchName '$(Build.SourceBranch)'
```